src/test/java/org/broadinstitute/consent/http/service/VoteServiceTest.java

```diff
@@ -293,8 +287,6 @@ public void testUpdateVotesWithValue_emptyList() throws Exception {
   public void testUpdateVotesWithValue_ClosedElection() throws Exception {
     when(electionDAO.findElectionsByIds(any())).thenReturn(List.of());
     Vote v = setUpTestVote(true, true);
-    when(voteDAO.findVoteById(anyInt())).thenReturn(v);
-    when(voteServiceDAO.updateVotesWithValue(any(), anyBoolean(), any())).thenReturn(List.of(v));
```

Comment:
These type of mocks were removed because they didn't contribute anything to the test, correct?

Reply:
This is correct - Mockito warns us with `UnnecessaryStubbingException` when we stub something that isn't actually exercised in the given test case.